apps/prairielearn/src/pages/partials/navbar.ejs

```diff
@@ -271,24 +271,35 @@
           %>
             <h6 class="dropdown-header">Effective user</h6>
 
-            <div class="dropdown-item-text d-flex flex-row justify-content-left ml-4 align-items-center">
-              <div class="px-0"><input type="email" class="form-control form-control-sm" id="navbar-user-effective-uid-input" placeholder="netid@illinois.edu" style="min-width:15em"></div>
-              <div class="pl-2"><button type="button" class="btn btn-primary btn-sm text-nowrap" id="navbar-user-effective-uid-button" disabled>Change UID</button></div>
-            </div>
+            <form class="form-inline dropdown-item-text d-flex js-effective-uid-form">
```

Comment:
The `flex-row` class was removed, which means that the button will be moved to below the input if the menu is too narrow (e.g., when a course is selected but not a course instance), instead of forcing the menu to increase its width. This was not the case before. Is this on purpose?

<img width="235" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/1004907/5a977da9-499c-4614-b566-7f100778e8bc">

Reply:
I added `flex-nowrap` in 5c99d61, which should resolve this.